.gitignore

```diff
@@ -47,6 +47,8 @@ dev/pr-deps/
 dist/
 docs/_site
 docs/api
+sql/docs/docs
```

Comment:
Sure, I can. I actually was wondering where I should put those.

Just to double check, would it be okay to simply put four (`sql/docs/mkdocs.yml`, `sql/docs/log4j.properties`, `sql/docs/gen-sql-markdown.py` and `sql/docs/create-docs.sh`) files into `sql/`?